tests/pytests/integration/runners/test_manage.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+
+@pytest.fixture
+def blackout_minion(salt_minion, salt_master, salt_run_cli):
+    with pytest.helpers.temp_file(
+        "top.sls",
+        """base:\n  '*':\n    - fnord""",
+        salt_master.pillar_tree.base.paths[0],
```

Comment:
```suggestion
        salt_master.pillar_tree.base.paths.write_path,
```
Should do the same.

Reply:
Actually, use `salt_master.pillar_tree.base.temp_file`

Reply:
Done. Hopefully tests pass tonight :joy: